kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -366,6 +458,12 @@
       },
     },
     methods: {
+      showTopicSizeWarningCard(content) {
```

Comment:
This makes me think that we need to properly update this method to handle to topic selection- for now I think it's getting lost in the way we are handling back route navigation.